stock_no_negative/i18n/fr.po

```diff
@@ -23,15 +23,15 @@ msgstr ""
 #: code:addons/stock_no_negative/models/stock_quant.py:38
 #, python-format
 msgid " lot '%s'"
-msgstr "lot%s"
+msgstr "lot '%s'"
```

Comment:
@alexis-via Could you remove changes in no_negative module from this PR ?

Reply:
Oups ! Well, this PR is closed now, so I don't think i'll fix it.